src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,21 @@
+package duke.commands;
+
+import duke.ui.Ui;
+import duke.tasks.TaskList;
+import duke.storage.Storage;
+import duke.exceptions.DukeException;
+
+/**
+ * Represents a command that can be run by Duke.
+ */
+public interface Command {
+    /**
+     * Parses the given argument string and executes the Command.
```

Comment:
if the method is really doing both parsing and executing the command, you may want to separate out the two functionalities.